sklearn/feature_extraction/text.py

```diff
@@ -1222,7 +1292,6 @@ def __init__(self, input='content', encoding='utf-8',
                  max_features=None, vocabulary=None, binary=False,
                  dtype=np.int64, norm='l2', use_idf=True, smooth_idf=True,
                  sublinear_tf=False):
-
```

Comment:
please try not to make any non-function modifications to the code; it makes it harder to review / in this case the newline should be there.